testing/tests/precompiles.rs

```diff
@@ -178,11 +178,12 @@ fn precompiles_tests() {
         .unwrap();
 
     gas_result.push(("get_actor_type".into(), res.msg_receipt.gas_used));
-    assert_eq!(res.msg_receipt.exit_code.value(), 0);
-    assert_eq!(
+    // FIXME
+    assert_eq!(res.msg_receipt.exit_code.value(), 33);
```

Comment:
get_actor_type precompile was removed, so we could remove the code and its test case